lib/cppcheck.cpp

```diff
@@ -560,6 +563,10 @@ unsigned int CppCheck::check(const std::string &path, const std::string &content
 
 unsigned int CppCheck::check(const FileSettings &fs)
 {
+    // TODO: move to constructor when CppCheck no longer owns the settings
```

Comment:
Yet another case highlighting how all the current messes I am working on are woven together...